### PR TITLE
Update to latest schema.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: templates/src/main/scala/lucuma/schemas/ObservationDB.scala [hunk 1]
```diff
@@ -16,27 +16,27 @@ import lucuma.core.math.BrightnessUnits._
...
-    type StepId              = Step.Id
-    type TargetId            = Target.Id
-    type TargetEnvironmentId = TargetEnvironment.Id
+    type AtomId           = Atom.Id
+    type ExecutionEventId = ExecutionEvent.Id
+    type ObservationId    = Observation.Id
+    type ProgramId        = String
+    type StepId           = Step.Id
+    type TargetId         = Target.Id
+    type VisitId          = Visit.Id
```
Comment (toddburnside) [inline templates/src/main/scala/lucuma/schemas/ObservationDB.scala:25]: I'd lost track. I didn't even remember we had anything like a TargetEnvironment.Id anymore.

**Review by toddburnside — APPROVED** [review]
LGTM

**Review by cquiroz — APPROVED** [review]
LGTM

File: templates/src/main/scala/lucuma/schemas/ObservationDB.scala (hunk 1 above)
Comment (rpiaggio) [reply templates/src/main/scala/lucuma/schemas/ObservationDB.scala:25]: At some point it was a thing.

**Pull request merged**